packages/client/scripts/default-deno-edge.ts

```diff
@@ -1,8 +1,8 @@
 class PrismaClient {
   constructor() {
     throw new Error(
-      `@prisma/client/deno/edge did not initialize yet. Please run "prisma generate --data-proxy" and try to import it again.
-In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues`,
+      `@prisma/client/deno/edge did not initialize yet. Please run "prisma generate" and try to import it again.
+      In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues`,
```

Comment:
Indentation is not part of the string on the previous line

Reply:
Nice catch, thanks

Reply:
Nice catch, thanks.